test-utils/src/lib.rs

```diff
@@ -64,6 +64,20 @@ pub async fn sign_compact<T: Signable + std::marker::Send>(signer: &Signers, sig
     return B512::try_from(signature_to_compact(&signed.signature).as_slice()).unwrap();
 }
 
+// TODO: figure out why this has different behavior than get_revert_string
```

Comment:
oh interesting, this seems like they use an enum and we revert with strings

https://github.com/FuelLabs/sway-libs/blob/master/libs/ownership/src/errors.sw#L3

I assume they have a setup like custom errors work, where the on-chain revert value is something sorta unintelligible and then with the use of the ABI using fuels-rs, they recover the enum variants and expose that to the user